pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -152,6 +156,11 @@ def test_names(self, data, time_locale):
 
     def test_is_leap_year(self, tz_naive_fixture):
         tz = tz_naive_fixture
+        if not IS64 and tz == tzlocal():
+            # https://github.com/dateutil/dateutil/issues/197
+            pytest.skip(
+                "tzlocal() on a 32 bit platform causes internal overflow errors"
+            )
```

Comment:
are we giving incorrect results in these cases?

Reply:
Running the 32 bit container locally it appears this still returns the correct result but hits the traceback in the dateutil issue

```
     File "pandas/_libs/tslibs/tzconversion.pyx", line 125, in pandas._libs.tslibs.tzconversion.Localizer.utc_val_to_local_val
      return utc_val + _tz_localize_using_tzinfo_api(
    File "pandas/_libs/tslibs/tzconversion.pyx", line 736, in pandas._libs.tslibs.tzconversion._tz_localize_using_tzinfo_api
      dt = _astimezone(dts, tz)
    File "pandas/_libs/tslibs/tzconversion.pyx", line 768, in pandas._libs.tslibs.tzconversion._astimezone
      return tz.fromutc(result)
    File "/root/virtualenvs/pandas-dev/lib/python3.8/site-packages/dateutil/tz/_common.py", line 144, in fromutc
      return f(self, dt)
    File "/root/virtualenvs/pandas-dev/lib/python3.8/site-packages/dateutil/tz/_common.py", line 261, in fromutc
      _fold = self._fold_status(dt, dt_wall)
    File "/root/virtualenvs/pandas-dev/lib/python3.8/site-packages/dateutil/tz/_common.py", line 196, in _fold_status
      if self.is_ambiguous(dt_wall):
    File "/root/virtualenvs/pandas-dev/lib/python3.8/site-packages/dateutil/tz/tz.py", line 254, in is_ambiguous
      naive_dst = self._naive_is_dst(dt)
    File "/root/virtualenvs/pandas-dev/lib/python3.8/site-packages/dateutil/tz/tz.py", line 260, in _naive_is_dst
      return time.localtime(timestamp + time.timezone).tm_isdst
  OverflowError: timestamp out of range for platform time_t
```